backend/editor/controllers/node_controller.py

```diff
@@ -0,0 +1,9 @@
+from ..graph_db import get_current_transaction
+
+
+async def delete_project_nodes(project_id: str):
```

Comment:
A comment to say it includes errors would be better (because I had to check how errors where reported in parser to verify we did not forgot them).

```suggestion
async def delete_project_nodes(project_id: str):
    """Remove all nodes for project.

    This includes entries, stopwords, synonyms and errors
    """
```